buzsaki_lab_to_nwb/yuta_code_visual_cortex/yutavcnwbconverter.py

```diff
@@ -0,0 +1,79 @@
+from pathlib import Path
+from datetime import datetime
+
+from nwb_conversion_tools import NWBConverter, NeuroscopeRecordingInterface, NeuroscopeLFPInterface
+
+from buzsaki_lab_to_nwb.yuta_code_visual_cortex.yutavcbehaviorinterface import YutaVCBehaviorInterface
+
+
+class YutaVCNWBConverter(NWBConverter):
+    """Primary conversion class for the SenzaiY visual cortex data set."""
+
+    data_interface_classes = dict(
+        NeuroscopeRecording=NeuroscopeRecordingInterface,
+        NeuroscopeLFP=NeuroscopeLFPInterface,
+        YutaVCBehavior=YutaVCBehaviorInterface,
+    )
+
+    def get_metadata(self):
+
```

Comment:
I would put text-based metadata in a YAML file

Reply:
I agree, this looks better now by separating code vs. big text blocks.